mac/sound.cpp

```diff
@@ -463,7 +463,7 @@ QString CSound::CheckDeviceCapabilities ( const int iDriverIdx )
          ( !( CurDevStreamFormat.mFormatFlags & kAudioFormatFlagIsPacked ) ) )
     {
         return tr ( "The audio input stream format for this audio device is "
```

Comment:
```suggestion
        return tr ( "The audio input stream format for this sound hardware is "
```

This suggestion sounds strange to me but would be needed if we follow the style guide.

Reply:
Yes, the trouble with style guide vocabulary rules is that don't always fit the context. I'd say it would read a bit better if it was ".. for your sound hardware" perhaps.

Reply:
`The stream format on the current input device is not %1. Please select another device or reconfigure.`

I really don't like `hardware`.  From Jamulus's perspective, it's just the audio source or audio sink.  We have no idea what's on the end of the device driver.

Reply:
We could maybe say in the guidelines that "device" is interchangeable with "hardware" if needed? The former is less newbie friendly, so would prefer "hardware" in most cases, but here perhaps "device" is more helpful maybe the OS refers to "devices" too?

Reply:
In Windows, click the speaker icon in the tooltray to pop open the the volume, then click the "^" to pop up the selector list -- it says "Select playback device".  So Windows users will understand "recording device" (for an input) and "playback device" (for an output).

Reply:
The overall control is "Open Sound settings", where "Output"->"Choose your output device" and "Input"->"Choose your input  device" live.

Reply:
So we could modify what we have in the guide so that "device" would be OK along with related terms:

"Device" to refer to audio hardware seen by the OS. "No device selected", "Input device”, "Choose the device".

“Audio interface” to refer to external devices. “If your audio interface comes with a driver”, “Your audio interface may be set to ‘monitor’”.

"Sound card" to refer to built-in audio devices. "Your computer's sound card".

“Sound hardware” refers generically to both internal and external devices. “Your sound hardware will introduce some latency”, “Most sound hardware can be used”.

Reply:
Well, this is `mac/sound.cpp`, so it should follow Apple MacOS conventions, rather than Microsoft Windows conventions.  We should follow OS language conventions in OS-specific messages.

Reply:
Ok. So "stick to naming conventions your OS uses" should be added to the google doc probably?

Reply:
@mulyaj to discuss this one today